Makefile

```
@@ -199,6 +199,13 @@ install-deps:
 	pip install --upgrade -r requirements.txt
 	python utils/install_actor_deps.py --actor=$(ACTOR)
 
+install-deps-fedora:
+	dnf install -y git findutils python3-virtualenv
```

Expecting this command should never be executed under root on real system, it will be better to do it like that (so in case of CI, ... - it will be done under the root, and in other case, people will now what they should do):
```suggestion
    if ! rpm -q git findutils python3-virtualenv; then \
        if ! dnf install -y git findutils python3-virtualenv; then \
            echo 'Please install the following rpms via the command: ' \
                'sudo dnf install -y git findutils python3-virtualenv'; \
            exit 1; \
        fi; \
    fi
```